src/Http/Response/StreamedResponse.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Baldinof\RoadRunnerBundle\Http\Response;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Basically a copy of Symfony's StreamedResponse,
+ * but the callback needs to be a Generator
+ */
+class StreamedResponse extends Response
+{
+    protected ?\Closure $callback = null;
+    protected bool $streamed = false;
+
+    private bool $headersSent = false;
+
+    public function __construct(\Generator $callback = null, int $status = 200, array $headers = [])
+    {
+        parent::__construct(null, $status, $headers);
+
+        if (null !== $callback) {
+            $this->setCallback($callback);
+        }
+    }
+
+    public function setCallback(\Generator $callback): static
```

Comment:
I think this should still accept a callable, then in the `sendContent` we can test if the callback returned a Generator or nothing. And if so consume and echo the generator